### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by hyl29**
- Removed pronouns to improve clarity (Replacing "her" with "Chantelle")

- Removed apostrophe "it's" to it is to increase formality 

- Added Chantelle's # at the end 

"This is a CMPT376 assignment where students must modify and improve other individuals' documentation"

### CODE REVIEW

**Comment by klusark** [comment]
Seems okay to me. Do people still use this code?

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ her what we need. Current info (as of 2014 June 11):
 - Fax:   604-464-5056
 - Email: cneff@coca-cola.com
 
-As she is often not in the office, it's best to call her cell.
+As Chantelle is often not in the office, it is best to call her cell (**604-830-4022**).
```
Comment (klusark) [inline README.md:32]: Well, the cell number is right above. Isn't it redundant to have it here?

Comment (hyl29) [reply README.md:32]: I am not sure if people still use this code but the 376 assignment wants us to improve the clarity of documents.
HMMMMMM. I put it that to emphasized the importance of the number.
We can just preserve the pronoun edit.

**Comment by modernNeo** [comment]
@klusark no one uses this code as it can no longer be deployed but still kept around for 376w purposes :smile:

**Review by modernNeo — APPROVED** [review]

**Pull request merged**